db/migrate/20210524124418_create_subjects.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSubjects < ActiveRecord::Migration[6.1]
+  def change
+    create_table :subjects do |t|
+      t.text :name
```

Comment:
@rubyboy-hub  Hi, the type of the column can be string, because the subject's name would be < 256 char . The text would have length 65555... something I forgot. what do you think?

Reply:
@rubyboy-hub  Same thing for standard.name

Reply:
changed to string instead of text